cognite/powerops/custom_modules/power_model_v1/data_models/views/shop_result/PriceProduction.view.yaml

```diff
@@ -0,0 +1,51 @@
+space: "{{powerops_models}}"
+externalId: PriceProduction
+name: PriceProdution
+description: TODO
+filter:
+  and:
+    - hasData:
+        - type: container
+          space: "{{powerops_models}}"
+          externalId: PriceProduction
+    - equals:
+        property:
+          - node
+          - space
+        value: "{{powerops_instance_space}}"
+    - equals:
+        property:
+          - node
+          - type
+        value:
+          externalId: PriceProduction
+          space: "{{powerops_type_space}}"
+implements: []
+version: "{{version}}"
+properties:
+  price:
+    container:
+      space: "{{powerops_models}}"
+      externalId: PriceProduction
+      type: container
+    containerPropertyIdentifier: price
+    name: price
+  production:
+    container:
+      space: "{{powerops_models}}"
+      externalId: PriceProduction
+      type: container
+    containerPropertyIdentifier: production
+    name: production
+  shopResult:
```

Comment:
The same shopResult object will appear multiple times in the array within the ShopPartialBidMatrixCalculationInput object. Why was shopResult here again?